modules/cpmso-checking-status.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * machine_management/cpmso-getting-started.adoc
+
+:_content-type: PROCEDURE
+[id="cpmso-checking-status_{context}"]
+= Checking the Control Plane Machine Set Operator status
+
+You can verify the status of the Control Plane Machine Set Operator and the existence of the `ControlPlaneMachineSet` custom resource (CR).
+//added "and the existence of the `ControlPlaneMachineSet` custom resource (CR)" as a result of the NotFound bullet below.
+
+.Procedure
+
+* Determine the state of the Operator by running the following command:
++
+[source,terminal]
+----
+$ oc get controlplanemachineset.machine.openshift.io cluster --namespace openshift-machine-api
+----
+
+** A result of `Active` indicates that the Operator is activated and functioning normally. No administrator action is required.
+
+** A result of `Inactive` indicates that the Operator is not activated, but a generated `ControlPlaneMachineSet` CR exists.
+
+** A result of `NotFound` indicates that there is no generated `ControlPlaneMachineSet` CR.
+
+.Next steps
+
+Before activating the Operator, you must ensure that a `ControlPlaneMachineSet` CR with the correct settings for your cluster exists. 
```

Comment:
Linked to: https://github.com/openshift/openshift-docs/pull/52802/files#r1028380991